snips/Dockerfile

```diff
@@ -5,12 +5,25 @@ FROM $BUILD_FROM
 COPY run.sh /
 COPY mosquitto.conf /etc/
 COPY customtts.sh /usr/bin
+COPY snips-entrypoint.sh /
 
-RUN chmod a+x /run.sh \
+ARG BUILD_FROM
+
+RUN echo nameserver 1.1.1.1 > /etc/resolv.conf \
+    && apt update \
+    && apt-get install -y dirmngr apt-utils apt-transport-https jq unzip supervisor mpg123 curl tzdata
+
+RUN if [ "$BUILD_FROM" = "homeassistant/amd64-base-ubuntu" ]; then bash -c 'echo "deb https://debian.snips.ai/stretch stable main" > /etc/apt/sources.list.d/snips.list' \
```

Comment:
```
ARG BUILD_ARCH
RUN if [ "$BUILD_ARCH" = "amd64" ]...
```

Reply:
Great, thanks. Wasn't sure what variables we would have defined.